k8srm-prototype/pkg/api/claim_types.go

```diff
@@ -53,66 +37,30 @@ type DeviceClassSpec struct {
 	// +required
 	DeviceType string `json:deviceType,omitempty`
```

Comment:
I am still not sure we NEED this, or if we do that it will be sufficient as a single string.  It feels like a list of "tags".

Could this be added later when we understand the need better?